<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetVigaSimplementeApoyada</class>
 <widget class="QWidget" name="WidgetVigaSimplementeApoyada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Longitud [cm]</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Posición apoyo izquierdo [cm]</string>
       </property>
       <property name="buddy">
        <cstring>txtPosicionApoyoIzquierdo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Posición apoyo derecho [cm]</string>
       </property>
       <property name="buddy">
        <cstring>txtPosicionApoyoDerecho</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtPosicionApoyoIzquierdo"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtPosicionApoyoDerecho"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
